### ISSUE TITLE
When I compile the code with TypeScript, the compiled code uses require statement and I get  " Must use import to load ES Module" error

### ISSUE THREAD

**Issue by SivaramShabari** [konvajs/konva#1196]
When I compile the code with TypeScript, the compiled code uses require statement and I get the same error.

TS code:

```
import Konva from "konva";
import { writeFileSync } from "fs";
let stage: any = new Konva.Stage({
	width: 500,
	height: 500,
	container: "container",
});
let layer = new Konva.Layer();

let text = new Konva.Text({
	x: 10,
	y: 10,
	height: 100,
	width: 400,
	text: "Hello World",
	fontSize: 60,
	fontFamily: "Calibri",
	fill: "green",
	draggable: true,
});
console.log(text.attrs.text, text.textArr[0].text);
if (text.attrs.text !== text.textArr[0].text) {
	while (text.attrs.text !== text.textArr[0].text) {
		text.setAttr("fontSize", text.attrs.fontSize - 1);
		console.log(text.attrs.fontSize);
	}
}
stage.add(layer);
layer.add(text);
let img = stage.toDataURL({ pixelRatio: 3, mimeType: "image/jpeg" });
let data = img.replace(/^data:image\/\w+;base64,/, "");
let buffer = Buffer.from(data, "base64");
writeFileSync("test.jpg", buffer);

```
Compiled Code:

```
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
const konva_1 = __importDefault(require("konva"));
const fs_1 = require("fs");
let stage = new konva_1.default.Stage({
    width: 500,
    height: 500,
});
let layer = new konva_1.default.Layer();
let text = new konva_1.default.Text({
    x: 10,
    y: 10,
    height: 100,
    width: 400,
    text: "Hello World",
    fontSize: 60,
    fontFamily: "Calibri",
    fill: "green",
    draggable: true,
});
console.log(text.attrs.text, text.textArr[0].text);
if (text.attrs.text !== text.textArr[0].text) {
    while (text.attrs.text !== text.textArr[0].text) {
        text.setAttr("fontSize", text.attrs.fontSize - 1);
        console.log(text.attrs.fontSize);
    }
}
stage.add(layer);
layer.add(text);
let img = stage.toDataURL({ pixelRatio: 3, mimeType: "image/jpeg" });
let data = img.replace(/^data:image\/\w+;base64,/, "");
let buffer = Buffer.from(data, "base64");
(0, fs_1.writeFileSync)("test.jpg", buffer);

```

Error:
![image](https://user-images.githubusercontent.com/60664289/138266423-11e499fa-e56b-4206-bf86-de529507a214.png)

_Originally posted by @SivaramShabari in https://github.com/konvajs/konva/issues/1126#issuecomment-948509381_

**Comment by omarryhan** [comment]
importing from 'konva/cmj' is breaking my types. (Next.js 12)

**Comment by lavrton** [comment]
How? Demo? Why do you import Konva in Next.js? Probably you should do it only on the client side.

**Comment by omarryhan** [comment]
I meant client side Next.js. (albeit it will be inevitably imported on the server when Next.js prerenders the given page).

I ended up ignoring it in the build step. Also, not using the special /cmj import anymore.

For anyone needing my solution, please tag me and I'll share it.

Thanks @lavrton

**Comment by lavrton** [comment]
The issue should be resolved with `konva@8.3.6`.
